### DISCUSSION TITLE
Usage of samples folder and cache

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#693] [General]
When cache is created , it is always loaded so that the original sampleset folder can be even deleted: is this true ?
I mean, nothing else is needed by GO when it loads organ cache file ? 
Of course i do know that if i make some changes in organ settings i have to update cache... Just to be sure i can take apart, for example on external hard disk, all samplesets and make them available only when i want to make some changes.
thanks

**Comment by oleg68** [comment]
GO may try to update cache (bad magic warnings and so on) on the next load, so I'd not recommend to do this.
